app/main.py

```diff
@@ -2,0 +1,17 @@
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "cp":
+        return
+
+    source_file = parts[1]
+    destination_file = parts[2]
+
+    if source_file == destination_file:
+        return
+
+    with open(
+            source_file, "r"
+    ) as file_in, open(
+        destination_file, "w"
+    ) as file_out:
+        file_out.write(file_in.read())
```

Comment:
The use of context managers to open files is correct and follows the task guidelines. Ensure that any exceptions during file operations are handled appropriately, although not explicitly required by the task.